### ISSUE TITLE
Hex View

### ISSUE THREAD

**Issue by mattbucknall** [thomasnordquist/MQTT-Explorer#280]
Thanks for MQTT Explorer - It is a great tool. I have a feature request:
Would it be possible to add a hex editor style view for viewing binary payloads? A 'hex' input type (in addition to raw/xml/json) would be handy too, for publishing MQTT messages with arbitrary binary content.

**Comment by Laro88** [comment]
@mattbucknall Agreed it would be nice, if you badly need it then you can get HEX from MQTT.fx
MQTT.fx is always 2bytes chunks x 8, this format is reasonably nice to look at.
7278 is the ascii letters "rx" from the log message.
![image](https://user-images.githubusercontent.com/12495700/72102741-01cad580-3328-11ea-8e3e-81323918557d.png)

**Comment by thomasnordquist** [comment]
Yes, this would be a nice feature.
